pkg/controller.v1/tensorflow/pod.go

```diff
@@ -240,6 +241,22 @@ func (tc *TFController) createNewPod(tfjob *tfv1.TFJob, rt, index string, spec *
 	return nil
 }
 
+func checkPodTTL(tfjob *tfv1.TFJob) bool {
+	if value, ok := tfjob.Annotations[TFJobWaitingWorkerAnnotation]; ok && value != "" {
+		podTTLAfterFinished, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return true
```

Comment:
It's important to log error when you decide to ignore it.

Reply:
Done